packaging/win32/installer.nsi

```diff
@@ -33,7 +33,7 @@
 !define LICENSE_FILEPATH "${PARSEC_FREEZE_BUILD_DIR}\LICENSE.txt"
 !define INSTALLER_FILENAME "parsec-${PROGRAM_VERSION}-${PROGRAM_PLATFORM}-setup.exe"
 
-!define WINFSP_INSTALLER "winfsp-1.7.20172.msi"
+!define WINFSP_INSTALLER "winfsp-1.8.20304.msi"
```

Comment:
so WinFSP 1.8 will only be provided to new users (given we test winfsp presence between 1.4 and 2.0), I guess it's ok

Reply:
Yes, until we implement a better update workflow for both winfsp and parsec :)